types/ssh2/index.d.ts

```diff
@@ -219,7 +219,8 @@ export interface Channel extends Duplex {
      */
     setWindow(rows: string, cols: string, height: string, width: string): void;
     signal(signalName: string): void;
-    exit(statusOrSignal: string, coreDumped: string, msg: string): void;
+    exit(status: number): void;
+    exit(signalName: string, coreDumped: boolean, msg: string): void;
```

Comment:
The docs indicate that the last two params are optional, so I think the definition should be like this:

```suggestion
    exit(signalName: string, coreDumped?: boolean, msg?: string): void;
```